pypeit/scripts/launch_identify.py

```diff
@@ -75,6 +75,12 @@ def main(args):
         lamplist = args.lamps.split(",")
     par['lamps'] = lamplist
 
+    # Extract binning information
+    binspec, binspat = parse.parse_binning(spec.get_meta_value(msarc.head0['F1'], 'binning'))
+
+    # Set the gain (read gain from headers)
+    _, _, _, _, _ = spec.get_rawimage(msarc.head0['F1'], args.det)
```

Comment:
If you're not catching anything that's returned, you can just run the method.  I.e., get rid of the `=` and everything to the left of it.